annotations.json

```
@@ -1,8 +1,132 @@
 {
   "version": 1,
   "classes": {
+    "net/minecraft/util/math/BlockBox": {
+      "methods": {
+        "encompass(Lnet/minecraft/util/math/BlockBox;)Lnet/minecraft/util/math/BlockBox;": {
+          "add": [
+            {
+              "desc": "Lorg/jetbrains/annotations/Contract;",
+              "values": {
+                "value": {
+                  "type": "string",
+                  "value": "!null -> this"
```

```suggestion
                  "value": "_ -> this"
```
You should only use `!null` if null is a valid input according to nullability annotations (params are not null by default in Minecraft). You've done this a lot

---

Fixed on next push